medical/models/medical_patient.py

```diff
@@ -36,12 +36,19 @@ class MedicalPatient(models.Model):
         compute='_compute_is_deceased',
     )
     marital_status = fields.Selection([
-        ('s', 'Single'),
-        ('m', 'Married'),
-        ('w', 'Widowed'),
-        ('d', 'Divorced'),
-        ('x', 'Separated'),
-        ('z', 'law marriage'),
+        ('S', 'Never Married'),
```

Comment:
Please leave all of the old selection keys in place for backward compatibility. The values can be modified as needed for greater alignment with FHIR, and new options are definitely welcome.